dub.sdl

```diff
@@ -0,0 +1,39 @@
+name "mir-stat"
+description "Dlang Statistical Package"
+
+authors "Ilya Yaroshenko" "John Michael Hall"
```

Comment:
You can place you at first here. It may be changed in the future depending on the author's contribution.